blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpMetricsIngestionServer.java

```diff
@@ -172,7 +172,7 @@ private void buildProcessingChains() {
 
         this.defaultProcessorChain = AsyncChain
                 .withFunction(typeAndUnitProcessor)
-                .withFunction(rollupTypeCacher)
+                //.withFunction(rollupTypeCacher)
                 .withFunction(batchSplitter)
```

Comment:
@GeorgeJahad We are not caching the rollup type for numeric and string metrics. But we will continue to cache the rollup type in the statsdProcessorChain. Does that make sense?